source/_integrations/matter.markdown

```diff
@@ -238,15 +228,18 @@ Tasmota supports Matter over IP on all ESP32 based devices (in experimental phas
 
 ### General recommendations
 
-- Using Thread-based Matter devices in Home Assistant requires Home Assistant OS (version 10 and above) because of kernel patches to solve routing issues. Not using HAOS (and thus the official Matter add-on) is at your own risk.
+- Using Thread-based Matter devices in Home Assistant requires Home Assistant OS (version 10 and above) because of kernel patches to solve routing issues. Not using HAOS (and thus the official Matter add-on) is at your own risk but we do provide tips and tricks how to run the Matter Server as docker container yourself, including all requirements to the host and networking on the [matter server repository](https://github.com/home-assistant-libs/python-matter-server/blob/main/README.md)
```

Comment:
```suggestion
- Thread-based Matter devices can only be run on Home Assistant OS version 10 and above. Using other installation types is not supported. However, we do have some documentation on how to run a [Matter server repository](https://github.com/home-assistant-libs/python-matter-server/blob/main/README.md) as docker container yourself. 
```

Reply:
well, a container install is supported since this month but with all precautions and warnings applied

Reply:
Ok. Let me try again.